pephubclient/exceptions.py

```diff
@@ -31,3 +31,11 @@ class PEPExistsError(BasePephubclientException):
     def __init__(self, message: Optional[str] = None):
         self.message = message
         super().__init__(self.message or self.default_message)
+
+
+class FileDoesNotExistError(BasePephubclientException):
```

Comment:
Can we just use pythons built-in [`FileNotFounderror`](https://docs.python.org/3/library/exceptions.html#FileNotFoundError)?